packages/mocks/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spinnaker/mocks",
-  "version": "1.0.6",
+  "version": "1.0.7",
```

Comment:
Is version increment done manually? Will be helpful if the reviewer can describe the release / publish process for `@spinnaker/mocks`.

Reply:
Right now this is manual. After this merges, you can go to the `mocks` directory and run `npm publish`, which will update the registry to match this version.

Reply:
Sounds good. Thanks.